cli/azd/pkg/project/framework_service_python.go

```diff
@@ -130,12 +130,7 @@ func (pp *pythonProject) Package(
 	if err := buildForZip(
 		packageSource,
 		packageDest,
-		buildForZipOptions{
-			excludeConditions: []excludeDirEntryCondition{
-				excludeVirtualEnv,
-				excludePyCache,
-			},
-		}); err != nil {
+		buildForZipOptions{}); err != nil {
```

Comment:
each `framework_service_language` should continue to provide a default `excludeConditions` for when no .zipignore is defined in the project.

Maybe you can call a method here from the `dotignore` package and pass the default excluding files.  The method will check if .zipignore exists. If no, it uses the default rules passed by the `framework_service_language`. If yes, it uses the rules from the .zipignore